### ISSUE TITLE
Svelte 5 Prettier Formatting Bug: Type Annotations in Snippets Malformed

### ISSUE THREAD

**Issue by sherlockdoyle** [sveltejs/prettier-plugin-svelte#467]
### Describe the bug

When using Svelte 5 with TypeScript and formatting code with Prettier and prettier-plugin-svelte, type annotations in snippet declarations are being malformed. Specifically, the arrow and return type of function types are being removed.

For example, the following code:

```svelte
<script lang="ts"></script>
{#snippet test(fun: () => void)}{/snippet}
```

becomes:

```svelte
<script lang="ts"></script>
{#snippet test(fun: ()}{/snippet}
```

### Reproduction

1. Create a new Svelte 5 project with TypeScript
2. Install prettier and prettier-plugin-svelte
3. Configure Prettier to use the Svelte plugin (in `.prettierrc` or `prettier.config.js`)
4. Add the following code to a `.svelte` file:
   ```svelte
   <script lang="ts"></script>
   {#snippet test(fun: () => void)}{/snippet}
   ```
5. Run Prettier formatting on the file
6. Observe that the type annotation is malformed

### System Info

```shell
- svelte.svelte-vscode Version: 109.0.3
- Node: 20.10.0
- npm: 10.2.3
- prettier: 3.3.3
- prettier-plugin-svelte: 3.2.6
- svelte: 5.0.0-next.257
```

### Severity

blocking an upgrade

**Comment by sherlockdoyle** [comment]
Workaround:

```svelte
<script lang="ts">
  type Fun = () => void;
</script>
{#snippet test(fun: Fun)}{/snippet}
```

**Comment by dummdidumm** [comment]
This bug was fixed in prettier-plugin-svelte 3.2.7, make sure to upgrade to the latest version of that package
